src/mesh/NodeDB.cpp

```diff
@@ -246,6 +246,31 @@ NodeDB::NodeDB()
         config.position.gps_mode = meshtastic_Config_PositionConfig_GpsMode_ENABLED;
         config.position.gps_enabled = 0;
     }
+#ifdef USERPREFS_FIXED_GPS
+    if (myNodeInfo.reboot_count == 1) { // Check if First boot ever or after Factory Reset.
+        meshtastic_Position FixedGPS = meshtastic_Position_init_default;
```

Comment:
Let's do `fixedGPS` to stick with camelcase for locals

Reply:
So drop `if (myNodeInfo.reboot_count == 1) {` ?

Reply:
I think Ben means change the name of the FixedGPS variable to fixedGPS

Reply:
Oh.....
No problem in changing to `fixedGPS` but since everything else in `userPrefs.h` has a `USERPREFS_` prefix, shouldn't be `USERPREFS_fixedGPS` ?

Reply:
down two lines: meshtastic_Position FixedGPS